src/OpenIddict.Security/OpenIddictExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.Extensions.Internal;
+using NWebsec.Middleware;
+using System;
+
+namespace Microsoft.AspNet.Builder {
+    public static class OpenIddictExtensions {
+        public static OpenIddictBuilder UseSecurityHeaders([NotNull] this OpenIddictBuilder builder, Action<IFluentCspOptions> cspOptions = null) {
```

Comment:
Well that depends on how specific you want to be. Calling it UseNWebsec ties it very closely to that library from a user's perspective and prevents us changing the implementation to another library. However, considering that is already tied to NWebsec via the IFluentCspOptions parameter, that could be a valid name too. If we had our own abstraction for setting headers I'd argue against a rename. In this case, what do you want?